### ISSUE TITLE
Show update and review dates under title on Playbook pages

### ISSUE THREAD

**Issue by johnwaterworth** [dxw/playbook#1135]
We tried in the past to encourage groups to regularly review and update content in the Playbook. Showing last updated and last reviewed dates on pages would help encourage this.

Always show the date that a page was last updated (last_modified in Netlify, I think). Underneath the title seems like the best place. Although it could go in footnote.

Also show an optional review date. Updating the page should remove the last review date. There should be some way to add or update a review date, when you've done a review.

**Comment by yndajas** [comment]
Yup - I don't think [this](https://playbook.dxw.com/contributing/managing-the-playbook/#review) is formally happening

> The owner of each section of the Playbook should lead a review of the sections they own at least every quarter.
>
> To help facilitate regular review, we should aim for Playbook content to include last updated and last reviewed dates.

**Comment by yndajas** [comment]
@johnwaterworth see #1138 and the preview here: https://deploy-preview-1138--laughing-payne-b9fbd2.netlify.app/

**Comment by davidwinter** [comment]
Should this also be expanded upon to include some automation and notifications around checking review dates, setting review periods, and who or which teams are responsible for reviewing content?

I'm not sure we could use this GDS tool out of the box, but I was thinking something similar to this: https://technology.blog.gov.uk/2020/09/25/keeping-tech-docs-up-to-date-with-daniel-the-manual-spaniel/
